packages/account/src/BiconomySmartAccount.ts

```diff
@@ -172,7 +229,9 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
       initCode: nonce.eq(0) ? this.initCode : '0x',
       callData: callData
     }
+
     userOp = await this.estimateUserOpGas(userOp, overrides)
+    Logger.log('userOp after estimation ', userOp)
```

Comment:
These Logger.log across all files should be called based on a debug flag in class constructor.... SDK should not print any logs on its own unless developer has enabled it using debug flag.

Reply:
so logger only logs if debug is enabled. this check is already there in log function

Reply:
From where is the Logger class getting the debug flag?